rs/async_utils/src/lib.rs

```diff
@@ -48,7 +48,11 @@ pub async fn shutdown_signal(log: Logger) {
 /// Recommended way of starting a TCP listener given a socket addr. The function
 /// will panic if it cannot start the listener, because the OS error can't be
 /// handled by the caller.
-pub fn start_tcp_listener(local_addr: std::net::SocketAddr) -> tokio::net::TcpListener {
+pub fn start_tcp_listener(
+    local_addr: std::net::SocketAddr,
```

Comment:
We could also remove the `ip_addr` config and only have the port in the config (or hardcode the port). We could just bind the server to `0.0.0.0`.

Same for transport.

Reply:
let's discuss and I can do it in a follow-up MR.